include/irr/config/BuildConfigOptions.h.in

```diff
@@ -0,0 +1,59 @@
+#ifndef __IRR_BUILD_CONFIG_OPTIONS_H_INCLUDED__
+#define __IRR_BUILD_CONFIG_OPTIONS_H_INCLUDED__
+
+#cmakedefine _IRR_DEBUG
+
+// architecture
+#cmakedefine __IRR_COMPILE_WITH_ARM_SIMD_
+
+// libraries
+#cmakedefine _IRR_COMPILE_WITH_GLI_
+
+// OS
+#cmakedefine _IRR_PLATFORM_WINDOWS_
+#cmakedefine _IRR_PLATFORM_LINUX_
+#cmakedefine _IRR_PLATFORM_OSX_
+
+// graphics API backend
+#cmakedefine _IRR_COMPILE_WITH_OPENGL_
+#cmakedefine _IRR_COMPILE_WITH_VULKAN_
+
+// loaders
+#cmakedefine _IRR_COMPILE_WITH_MITSUBA_SERIALIZED_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_MTL_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_OBJ_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_STL_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_PLY_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_BAW_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_GLTF_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_JPG_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_PNG_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_TGA_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_OPENEXR_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_GLI_LOADER_
+#cmakedefine _IRR_COMPILE_WITH_BIN_LOADER_
+
+// writers
+#cmakedefine _IRR_COMPILE_WITH_STL_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_PLY_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_BAW_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_GLTF_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_TGA_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_JPG_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_PNG_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_OPENEXR_WRITER_
+#cmakedefine _IRR_COMPILE_WITH_GLI_WRITER_
+
+// compute interop
+#cmakedefine _IRR_COMPILE_WITH_OPENCL_
+#cmakedefine _IRR_COMPILE_WITH_CUDA_
+#cmakedefine _IRR_BUILD_OPTIX_
+
+// extra config
+#cmakedefine __IRR_FAST_MATH
+#cmakedefine _IRR_EMBED_BUILTIN_RESOURCES_
+
+// TODO: This has to disapppear from the main header and go to the OptiX extension header + config
+#cmakedefine OPTIX_INCLUDE_DIR "@OPTIX_INCLUDE_DIR@"
+
+#endif //__IRR_BUILD_CONFIG_OPTIONS_H_INCLUDED__
```

Comment:
wtf... there's no such file anymore!? this is `irr` not `nbl`

Reply:
buggy and irrelevant, resolving